pkg/cloudprovider/suite_test.go

```diff
@@ -314,6 +327,23 @@ var _ = Describe("CloudProvider", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(isDrifted).To(BeTrue())
 		})
+		It("should return drifted if the securitygroup is not valid", func() {
+			node := coretest.Node(coretest.NodeOptions{
```

Comment:
Can this test be based on a machine now that we are relying on Machines instead of nodes here?